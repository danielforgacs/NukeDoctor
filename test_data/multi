Node {
 1
 name KEEP_THIS_0001
 3
}
set arg1 arg2
Node {
 name KEEP_THIS_0002
}
NodeB {
 1
 name KEEP_NAME_ONLY_0001
 3
 4
 5
}
push arg1
IgnoreA {
 name KEEP_NAME_ONLY_0002
}
IgnoreB {
 data
 name KEEP_NAME_ONLY_0003
 3
}
