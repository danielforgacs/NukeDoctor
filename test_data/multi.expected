Node {
 1
 name KEEP_THIS_0001
 3
}
Node {
 name KEEP_THIS_0002
}
NodeB {
 name KEEP_NAME_ONLY_0001
}
IgnoreA {
 name KEEP_NAME_ONLY_0002
}
IgnoreB {
 data
 name KEEP_NAME_ONLY_0003
 3
}
